### DISCUSSION TITLE
Mods Home Row Cluster issues

### DISCUSSION THREAD

**Discussion by flipjs** [manna-harbour/miryoku#11] [Q&A] [answered]
I love the mods on home row and I really like to make it work for me. The issues I'm having is when Im typing sentences, mods+alphas trigger frequently. Its either alphabet letters will be converted into non-english letters or it will trigger some shortcuts in the app Im on.

Its probably not just me having this issues I guess. Also, I am a slow typist not to mention Im also new at ortholinear layout. I also changed the layout to qwerty. Not sure if that would matter.

Do you have any suggestion on how I can make this work? Maybe the mods on home row can be combined with the thumb keys to activate them?

I really love your work on this and thank you for sharing this with us.

**Comment by flipjs** [comment]
Here's one example that just happened: Typing "**at**" would trigger **Super-T** (Cmd-T on OS X). If Im inside a browser, that would create and switch to a new Tab window.

**Comment by flipjs** [comment]
I changed the order of the modifier keys and that helps a lot to some degree. I have a weak pinky and command key (L_GUI) is heavily used in OS X for shortcuts. So I guess I just have to tweak it. Thanks for this fantastic layout!

**Comment by blueyed** [comment, marked as answer]
JFI: see https://precondition.github.io/home-row-mods#using-home-row-mods-with-qmk (and the whole article) for more information/details, in particular the idea of disabling (or rather adjusting) rollovers on the same side.
